packages/scripts/scripts/build.js

```diff
@@ -1,39 +1,24 @@
-#!/usr/bin/env node
 const spawn = require('cross-spawn');
-const { getEnv } = require('./utils/env');
-const { hereRelative, resolveBin } = require('./utils/path-resolver');
-const { printSeparator } = require('./utils/log');
+const { hereRelative, resolveBin } = require('../utils/path-resolver');
 
 const rimraf = resolveBin('rimraf');
 const webpack = resolveBin('webpack');
 
-printSeparator('CONFIGURATION');
+module.exports = function build(env) {
+	// remove old builds folder
+	const rmResult = spawn.sync(
+		rimraf,
+		['./build', './dist'],
+		{ stdio: 'inherit', env }
+	);
+	if (rmResult.status === 0) {
```

Comment:
if it fails, it will stop I guess? Is this check really useful?

Reply:
We spawn a child process, so its failure doesn't impact the current process.
That's why in `index.js` we call process.exit with the child process status